client/requestUtil.js

```diff
@@ -150,17 +156,41 @@ RequestUtil.prototype.parseAWSResponse = function (bytes) {
     sslEnabled: true,
     useDualstack: true
   })
-  return {s3, postData, expiration, bucket, region}
+  const SQS = new awsSdk.SQS({
+    credentials: new awsSdk.Credentials({
+      accessKeyId: credentials.accessKeyId,
+      secretAccessKey: credentials.secretAccessKey,
+      sessionToken: credentials.sessionToken
+    }),
+    endpoint: `https://sqs.${region}.amazonaws.com`,
+    maxRetries: S3_MAX_RETRIES,
+    region: region,
+    sslEnabled: true
+  })
+  const SNS = new awsSdk.SNS({
+    credentials: new awsSdk.Credentials({
+      accessKeyId: credentials.accessKeyId,
+      secretAccessKey: credentials.secretAccessKey,
+      sessionToken: credentials.sessionToken
+    }),
+    endpoint: `https://sns.${region}.amazonaws.com`,
+    maxRetries: S3_MAX_RETRIES,
+    region: region,
+    sslEnabled: true
+  })
+
+  return {s3, postData, expiration, bucket, region, SQS, SNS}
```

Comment:
minor: capitalization on `SQS` vs `sqs`, in this repo we're using lowercase for instances and SQS for class names / constructors. can you adjust for consistency with `s3`?